datetimeparser/datetimeparser.py

```diff
@@ -3,17 +3,17 @@
 """
 
 __all__ = ['parse', '__version__', '__author__']
-__version__ = "0.13.4"
+__version__ = "0.14.4"
```

Comment:
Please fix version format: Major.minor.dev. For that update it would probably be better to just increment dev version.